docs/cli.md

```diff
@@ -0,0 +1,217 @@
+<!--
+Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Model Analyzer CLI
+
+Use the `-h` flag to view a description of the Model Analyzer's command line
+interface.
+
+```
+$ model-analyzer -h
+usage: model-analyzer [-h] [-q] [-v] [-m {online,offline}] {profile,analyze,report} ...
+
+positional arguments:
+  {profile,analyze,report}
+                        Subcommands under Model Analyzer
+    profile             Run model inference profiling based on specified CLI or config options.
+    analyze             Collect and sort profiling results and generate data and summaries.
+    report              Generate detailed reports for a single config
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -q, --quiet           Suppress all output except for error messages.
+  -v, --verbose         Show detailed logs, messages and status.
+  -m {online,offline}, --mode {online,offline}
+                        Choose a preset configuration mode.
+```
+
+Options like `-q` and `-v` are global and apply to all model analyzer
+subcommands. 
+
+## Model Analyzer Subcommands
+
+The Model Analyzer's functionality is split across 3 separate subcommands. Each
+subcommand has its own CLI and config options. Some options are required for
+more than one subcommand (e.g. `--export-path`). See the [Configuring Model
+Analyzer](./config.md) section for more details on configuring each of these
+subcommands.
+
+## Subcommand: `profile`
+
+The `profile` subcommand allows the user to run model inferences using perf
+analyzer, and collect metrics like throughput, latency and memory usage.
+
+```
+$ model-analyzer profile -h
```

Comment:
This is an excellent example in how the documentation will go out of sync from the product. Do not dump the output from the help, when the user can just enter help on the command line. Just describe the subcommand, general use guide, and some example, common command lines. For example, I should see here what the minimum command line I need to make this work.